AndroidManifest.xml

```diff
@@ -60,6 +60,7 @@
             android:label="@string/app_label"
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit